storage/tianmu/handler/ha_my_tianmu.cpp

```diff
@@ -25,7 +25,26 @@
 namespace Tianmu {
 namespace handler {
 
-enum class TIANMUEngineReturnValues { LD_Successed = 100, LD_Failed = 101, LD_Continue = 102 };
+enum class TianmuEngineReturnValues {
+  kLoadSuccessed = 0,
+  kLoadFailed,
+  kLoadContinue,
+};
+
+bool AtLeastOneTIANMUTableInvolved(LEX *lex) {
+  for (TABLE_LIST *table_list = lex->query_tables; table_list; table_list = table_list->next_global) {
+    TABLE *table = table_list->table;
+    if (core::Engine::IsTianmuTable(table))
+      return true;
+  }
+  return false;
+}
+
+bool ForbiddenMySQLQueryPath([[maybe_unused]] LEX *lex) {
```

Comment:
Maybe `(LEX *lex [[maybe_unused]])` is a better one.

Reply:
Fixed!